src/pages/projects/projects.tsx

```diff
@@ -3,14 +3,19 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
+import { Route, Switch } from 'react-router-dom';
 import ProjectsTable from '../../containers/projects-table/projects-table';
+import WBSDetails from '../wbs-details/wbs-details';
 import './projects.module.css';
 
 const Projects: React.FC = () => {
   return (
     <div>
       <h1>This is the Projects Page</h1>
-      <ProjectsTable />
+      <Switch>
+        <Route path="/projects/:wbsNum" component={WBSDetails} />
+        <Route path="/projects" component={ProjectsTable} />
```

Comment:
Will this route be in conflict with the route in App.tsx? Or is it because that this route is rendered within projects.tsx, so it only routes when both the path is /projects and the component that is currently rendered is projects.tsx?

Reply:
I don't think there's any conflict, and I think they do it the same way in their example: https://reactrouter.com/web/example/nesting